src/routers/Gateway.sol

```diff
@@ -79,14 +125,52 @@ contract ConnectorGateway {
         emit Deny(user);
     }
 
+    function pause() external onlyPauseAdmin {
+        paused = true;
+        emit Pause();
+    }
+
+    function unpause() external onlyPauseAdmin {
+        paused = false;
+        emit Unpause();
+    }
+
+    function scheduleRely24hr(address spell) internal {
```

Comment:
I would replace 24hr and 48hr with short and long, and make these immutable parameters filed on the constructor. That way we can update these times in the future by swapping the gateway, but the transaction naming doesn't change.

Reply:
I would also change `spell` to `admin` or something similar. This doesn't need to be a spell, in theory it could just as well be used to rely an individual end account that makes a small change, or a new contract that has some logic that ties into the gateway.